src/main/java/nu/studer/gradle/rocker/RockerPlugin.java

```diff
@@ -63,16 +61,7 @@ public void apply(Project project) {
     }
 
     private SourceSetContainer getSourceSets(Project project) {
-        if (isAtLeastGradleVersion("7.1")) {
-            return project.getExtensions().getByType(JavaPluginExtension.class).getSourceSets();
-        } else {
-            return getSourceSetsDeprecated(project);
-        }
-    }
-
-    @SuppressWarnings("deprecation")
-    private SourceSetContainer getSourceSetsDeprecated(Project project) {
-        return project.getConvention().getPlugin(org.gradle.api.plugins.JavaPluginConvention.class).getSourceSets();
+        return project.getExtensions().getByType(SourceSetContainer.class);
     }
```

Comment:
@jprinet Can you explain why you were able to remove this entire logic for old Gradle versions?